src/tests/ftest/datamover/dst_create.py

```diff
@@ -225,7 +223,7 @@ def verify_cont_prop(self, cont, prop_list, api):
         # Make sure each property matches
         for prop_idx, prop in enumerate(prop_list):
             # This one is not set
-            if api == "DFS" and "OID" in prop_list[prop_idx]:
+            if api == "DFS" and "OID" in str(prop["value"]):
```

Comment:
This is trying to skip `'description': 'Highest Allocated OID'`
```suggestion
            if api == "DFS" and "OID" in str(prop["description"]):
```

Reply:
ah I see. Thanks for taking a look!